magma/inputgenerator.py

```diff
@@ -272,6 +272,10 @@ def _match_argument_file(self, arg_obj, run_obj):
                 #end if
             #end for
             gather = getattr(arg_obj, 'gather', 0)
+            # If not gather look for gather_input
+            if not gather:
```

Comment:
Is an input of 0 a valid input for `gather`?

Reply:
for the part where you build a MetaWorkflowRun, ``gather: 0`` would be ignored as I have ``if gather:`` as a condition before registering it in the workflow object. So yea, ``0`` is not a valid input.